nipype/interfaces/afni/preprocess.py

```diff
@@ -44,11 +53,10 @@ class BlurToFWHM(AFNICommand):
     >>> blur = afni.preprocess.BlurToFWHM()
     >>> blur.inputs.in_file = 'epi.nii'
     >>> blur.inputs.fwhm = 2.5
-    >>> blur.cmdline #doctest: +ELLIPSIS
+    >>> blur.cmdline #doctest: +ELLIPSIS +IGNORE_UNICODE
```

Comment:
For python 3 this is not necessary. But, when using ```unicode_literals``` all the strings become unicode, and in python 2 ```repr(str)``` always has the ```u'``` prefix. This breaks doctests.

For this reason, to make doctests work in python 2 I added the doctest-ignore-unicode package, which requires that flag before any string doctest.